lib/messages.js

```diff
@@ -637,29 +669,33 @@ const LinearizerUpdate = {
     c.fixed32.preencode(state, m.key)
     c.uint.preencode(state, m.length)
     c.uint.preencode(state, m.batch)
-    InternalViews.preencode(state, m.internalViews)
+    c.uint.preencode(state, m.systemLength)
+    c.uint.preencode(state, m.encryptionLength)
```

Comment:
do you need encryptionLength here now since its tracked in system like the other views?